src/Components/Components/src/Rendering/Renderer.cs

```diff
@@ -423,6 +432,7 @@ private void ProcessRenderQueue()
             {
                 // Ensure we catch errors while running the render functions of the components.
                 HandleException(e);
+                return;
```

Comment:
This was a bug. After an exception we should stop everything immediately and not run anything else, like the code after this try..catch..finally block.